samples/ccr/start.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+set -e
+
+# Start metrics store
+docker-compose -f ./docker-compose-metricstore.yml up -d --remove-orphans
+
+# Start Elasticsearch
+docker-compose up -d --remove-orphans
+
+printf "Waiting for clusters to get ready "
+
+# Wait until ES is up
+ALL_CLUSTERS_READY=false
+
+while ! $ALL_CLUSTERS_READY; do
+    curl -ks -u admin:admin https://localhost:9200 -o /dev/null && curl -ks -u admin:admin https://localhost:9201 -o /dev/null && ALL_CLUSTERS_READY=true || (printf "." && sleep 5)
+done
+
+echo
+
+# Configure the seed nodes on follower cluster
+echo "Configure remotes on follower"
+curl -o /dev/null -H 'Content-Type: application/json' -k -u admin:admin -X PUT https://localhost:9201/_cluster/settings -d @- <<-EOF
+    {
+    "persistent" : {
+        "cluster" : {
+        "remote" : {
+            "source" : {
+            "seeds" : [
+                "localhost:9300"
```

Comment:
Shouldn't this be IP of the leader cluster node?

Reply:
this script sets up both clusters locally using docker. This is the leader's transport port i.e. 9300.